server/routes/charge-card.js

```diff
@@ -1,10 +1,10 @@
-var express = require('express');
-var Stripe = require('stripe')
-var router = express.Router();
-var bodyParser = require('body-parser')
+const express = require('express');
+const Stripe = require('stripe')
+const router = express.Router();
+const bodyParser = require('body-parser')
 
 const stripe = new Stripe('sk_test_51IweHkKvAxvZ5kVetx3UiymBoyjtPixhy4q72vX0lh5MaPQfI9r9zilffbnSnOXRsfbJKbYSNQAkTCB9vPWyTBOO00WrHtvZsQ');
-var jsonParser = bodyParser.json()
+const jsonParser = bodyParser.json()
```

Comment:
I think we can store the above stripe id in an env file as well

Reply:
Our API key is already exposed, I don't think it matters anymore. Just make sure to hide the new API key hen we launch